wp-content/mu-plugins/the-world-site-config/configs/global/global-plugins.php

```diff
@@ -9,11 +9,17 @@
   'co-authors-plus/co-authors-plus.php',
   'custom-post-type-ui/custom-post-type-ui.php',
   'disable-comments/disable-comments.php',
+  'PATCH-external-media-without-import/external-media-without-import.php',
```

Comment:
The only way to get audio upload to work as intended was to remove all the bits about Height, width and mime type to support this. So I made a copy of the module to track that its been edited.

(Wordpress doesn't have a default patching mechanism)